base/number.jl

```diff
@@ -112,10 +112,36 @@ copysign(x::Real, y::Real) = ifelse(signbit(x)!=signbit(y), -x, +x)
 conj(x::Real) = x
 transpose(x::Number) = x
 ctranspose(x::Number) = conj(x)
-inv(x::Number) = one(x)/x
 angle(z::Real) = atan2(zero(z), z)
 
 """
+    inv(x)
+
+Return the multiplicative inverse of `x`, such that `x*inv(x)` or `inv(x)*x`
+yields [`one(x)`](@ref) (the multiplicative identity).
```

Comment:
As @MikaelSlevinsky mentioned this isn't true. i.e. `x*inv(x)` or `inv(x)*x` does *not* yield `one(x)`

 ```
julia> x=49.0

julia> inv(x)*x
0.9999999999999999

julia> one(x)
1.0

```